bin/bios-baseline.sh

```diff
@@ -100,12 +100,29 @@ function ilo_enable_ipmitool {
     fi
 }
 
+# COMPATIBLE VENDOR(S): HPE
+function ilo_disable_dhcp_shared_nic() {
+    local actual
+    local error
+    local expected='DHCPEnabled=False'
+    actual=$(ilorest --nologo list DHCPv4/DHCPEnabled --selector EthernetInterface --filter Name='Manager Shared Network Interface')
+    if [[ "$actual" = *"$expected"* ]] ; then
```

Comment:
You have no way to tell if the ilorest command actually failed here and we're not running with `set -e`.